test/proto/BUILD

```diff
@@ -19,11 +19,8 @@ envoy_proto_library(
 envoy_proto_library(
     name = "bookstore_proto",
     srcs = [":bookstore.proto"],
-    external_deps = [
-        "api_httpbody_protos",
-        "http_api_protos",
-        "well_known_protos",
-    ],
+    external_cc_proto_deps = ["@googleapis//:api_httpbody_protos"],
```

Comment:
If here is the only instance probably fine for now, but can we have this proto_deps/cc_proto_deps map somewhere in `.bzl` files and automatically inject them in envoy_proto_library?